src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -4,14 +4,17 @@
 
   <PropertyGroup>
     <Description>Various helpers for writing tests that use ASP.NET Core.</Description>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Can we make this improvement after this conversion? I say this because this is a top repo on which other repo conversions depend on, so wondering if we can make this improvement later to unblock the conversion